variantrule/Modnar; Character Points.json

```diff
@@ -0,0 +1,489 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "Giddy",
+				"abbreviation": "Brew",
```

Comment:
Similarly, you might want to make this something mode identifiable. Not necessary, though.